_pages/business-units/18f/projects-partners/collaboration-tools.md

```diff
@@ -0,0 +1,168 @@
+
+<details>
+<summary>Compliance and records considerations</summary>
+* GitHub is ATO’d for use in GSA and has [a FedRAMP Agency Authorization at the LI-SaaS impact level](https://marketplace.fedramp.gov/#/product/github-enterprise-cloud). GitHub operates their own data centers.
+* All communication in GitHub is considered a record and conforms to GSA’s record management policy.
+</details>
+
+### Remote Whiteboarding
+
+**[Mural]({{site.baseurl}}/murally/)** is an online collaborative whiteboard tool. It allows teams to collectively generate ideas with sticky notes and sorting them around as if they were in the same room. You don’t need an account to participate and you can access the tool from any browser.
+
+<details>
+<summary> Compliance and records considerations</summary>
+* Mural is hosted on Azure commercial public cloud. It has a GSA ATO but doesn’t have a FedRAMP authorization or anything in process.
+* Mural generally wouldn’t be used to store “records” but instead to organize information for discussion purposes. GSA Records Officer has determined that “this product likely creates a number of record types. However, the specific types of records created will depend on the context in which this product is used.” As such, be sure to export and archive information from Mural periodically.
+* Truly low-impact: Mural shouldn’t be used to store anything confidential or authoritative. Instead, use it  for ephemeral organization of information and production visualizations. Non-GSA participants are invited to use the service anonymously.
+</details>
+ 
+## Helping partners make the most of these new tools
+ 
+Here are some recommendations for how to level up our partners' skills:
+
+### Provide a training or learning materials
+* Use or adapt Greg Walker’s [Slack training](https://gsa-tts.slack.com/files/U0KHZ7BE1/F82812ZJ6/Notes_from_Slack_training).
+* There are some useful Github-related resources listed in [this Slack thread](https://gsa-tts.slack.com/archives/CHZLJBWCV/p1559759830000500) or [this Slack thread](https://gsa-tts.slack.com/archives/C04KL9ZN2/p1571357280118500).
+
+### Learn by doing/watching/failing
+* Introduce product features as needed on calls. 
+* Let the partner play around with the tools in a low-risk environment—let them fail and “break” things. It’s a great way to learn! 
+
+### Develop shared conventions with your partner
+* Consider creating a “how we work” document detailing your team conventions (e.g. task labels, how you title a story, etc). It’s a lot easier to remind folks of the process when you have a document to point them to.
+ 
+ 
+## Helping partners transition off collaboration tools
+* **Important!** Don’t wait until the end of a project to think about this!
```

Comment:
If it's that important, maybe we should find a way to mention it earlier?

Reply:
Agreed. Added a summary above in: 
> Setting up tools with partners